vnext/Microsoft.ReactNative/Fabric/platform/react/renderer/textlayoutmanager/WindowsTextLayoutManager.cpp

```diff
@@ -513,7 +513,7 @@ LinesMeasurements TextLayoutManager::measureLines(
   return lineMeasurements;
 }
 
-Float TextLayoutManager::baseline(
+/* Float TextLayoutManager::baseline(
```

Comment:
How come this is being commented out? If the method was deleted upstream, we can delete here as well.